Sources/Router/Views/RouteViewIdView.swift

```diff
@@ -0,0 +1,16 @@
+import SwiftUI
+
+@available(iOS 13, macOS 10.15, *)
+struct RouteViewIdView<Content: View>: View, Equatable {
```

Comment:
What's this for?

Reply:
It's the reason PR reviews exist 😉 Thanks for noticing, it's a leftover from an earlier attempt. I've removed it.